scripts/ci/qemu-run.sh

```diff
@@ -52,8 +64,12 @@ case "$2" in
 	vnc)
 		qemu-system-x86_64 $q35_params -nographic -vnc :0
 		;;
+	install_ubuntu)
+		qemu-system-x86_64 $q35_params -cdrom qemu-data/ubuntu.iso
```

Comment:
Where I should get `qemu-data/ubuntu.iso` ? When I'm running this option I get  

```
qemu-system-x86_64: -drive file=qemu-data/hdd.qcow2,if=virtio: Could not open 'qemu-data/hdd.qcow2': No such file or directory
```

Reply:
I simply used 22.04.3 standard desktop installer and installed manually.
I wanted to use autoinstaller from https://github.com/Dasharo/preseeds/blob/main/ubuntu/create_image.sh but it seems it has worked for previous ISO version only. I wanted to focus on SB testing, not on OS installation testing here, so moved this task for later.